code/game/machinery/telecomms/presets.dm

```diff
@@ -106,11 +106,11 @@
 /obj/machinery/telecomms/bus/preset_four
 	id = "Bus 4"
 	network = "tcommsat"
-	autolinkers = list("processor4", "engineering", "common")
+	autolinkers = list("processor4", "engineering", "common", "ai private")
```

Comment:
do you know what this does?

Reply:
i forgot to remove this when i realized i didnt need to make a new server